os/board/rtl8720e/src/component/wifi/wifi_interactive_mode.c

```diff
@@ -811,12 +839,10 @@ int8_t cmd_wifi_on(WiFi_InterFace_ID_t interface_id)
 	wifi_show_setting(WLAN0_NAME, &setting);
 
 #if CONFIG_LWIP_LAYER
-		uint8_t *mac = LwIP_GetMAC(0);
-		nvdbg("\n\r  MAC => %02x:%02x:%02x:%02x:%02x:%02x", mac[0], mac[1], mac[2], mac[3], mac[4], mac[5]);
-		nvdbg("\n\r  IP  => %d.%d.%d.%d", ip[0], ip[1], ip[2], ip[3]);
-		netlib_setmacaddr(CONFIG_WIFIMGR_STA_IFNAME, mac);
+	uint8_t *mac = LwIP_GetMAC(0);

```

Comment:
Please consider declaring this variable at the beginning of the function under the Config check condition -
```
        int ret;
#if CONFIG_LWIP_LAYER
        uint8_t *mac;
#endif
```

Reply:
@amogh-samsung Sorry Amogh. But it's chipset vendor code so that we don't recommend TizenRT coding rules. Let's keep this as Realtek wants.